src/fixate/reporting/csv.py

```diff
@@ -367,6 +377,7 @@ def register_csv():
     pub.subscribe(writer.reporting.sequence_complete, "Sequence_Complete")
     pub.subscribe(writer.reporting.user_wait_start, "UI_block_start")
     pub.subscribe(writer.reporting.user_wait_end, "UI_block_end")
+    pub.subscribe(writer.reporting.discover_visa, "discover_visa")
```

Comment:
Rename this message to reflect its function rather than where we're calling it from. "driver_open" maybe?

Reply:
I'll rename it to "driver_open" and change the "VISA" string in the csv writer to "DRIVER"